pkg/executor/analyze_utils.go

```diff
@@ -66,7 +66,7 @@ func getAnalyzePanicErr(r interface{}) error {
 		}
 		return err
 	}
-	return errAnalyzeWorkerPanic
+	return errors.Trace(errAnalyzeWorkerPanic)
```

Comment:
the error stack should be binded automatically?
i think we dont need to use the Trace to wrap it

Reply:
In fact, I find this is not automatically and the stack of this error is wrong so that i have to trace it by the hand.